lib/server.ts

```diff
@@ -335,8 +340,8 @@ export function createApp(devJsonPath: string, opts: DevOptions): Express.Applic
   return app;
 }
 
-function initializeGateway(gatewayUrl: string, isLocalPath: boolean, opts: DevOptions, devJsonPath: string) {
-  gatewayInitPromise = setupGateway(gatewayUrl, isLocalPath, opts, devJsonPath)
+function initializeGateway(gateway: GatewayConfigObject, isLocalPath: boolean, opts: DevOptions, devJsonPath: string) {
+  return setupGateway(gateway, isLocalPath, opts, devJsonPath)
```

Comment:
Following up on previous comment, we don't need the return. Should be as it was before

gatewayInitPromise = setupGateway